### PR TITLE
help: document the usage of addSystemSynthDefs

### PR DESCRIPTION
**Pull request by telephon**
## Purpose and Motivation

Documentation for addSystemSynthDefs

### CODE REVIEW

File: HelpSource/Guides/Non-Realtime-Synthesis.schelp
Comment (telephon) [reply HelpSource/Guides/Non-Realtime-Synthesis.schelp:188]: OK, done, I hope it is a little clearer. I didn't have the time to go into more detail.

**Review by mtmccrea — APPROVED** [review]
This is great, thanks @telephon for going the extra step and adding a help file for `SystemSynthDefs`!

File: HelpSource/Guides/Non-Realtime-Synthesis.schelp
Comment (mtmccrea) [reply HelpSource/Guides/Non-Realtime-Synthesis.schelp]: Great! 
Just a note that it looks like you updated _Score.schelp_ to link `SystemSynthDefs`, but forgot to update the matching text here, so I've gone ahead and added that.

Thanks!

**Comment by mtmccrea** [comment]
Merge note: pls squash

File: HelpSource/Classes/SystemSynthDefs.schelp
Comment (capital-G) [inline HelpSource/Classes/SystemSynthDefs.schelp:50]:
```suggestion
method::maxTempDefNames
```

File: HelpSource/Classes/SystemSynthDefs.schelp
Comment (capital-G) [inline HelpSource/Classes/SystemSynthDefs.schelp:58]: Some excessive newlines - not a problem in general, but I think it would be cleaner to not have them.

**Review by capital-G — DISMISSED** [review]
I think the `:;` needs a fix, afterwards it can be merged.

File: HelpSource/Classes/SystemSynthDefs.schelp
Comment (capital-G) [inline HelpSource/Classes/SystemSynthDefs.schelp:58]: Some excessive newlines - not a problem in general, but I think it would be cleaner to not have them.

**Comment by mtmccrea** [comment]
@capital-G thanks for the change, I think you need to accept/dismiss the review in order to merge

**Comment by mtmccrea** [comment]
aside: I wonder if we can have CI render the help docs in order to catch these kinds of things (I think this would have caused a warning when rendering)

**Comment by capital-G** [comment]
@mtmccrea Actually a concerns was not fixed, see https://github.com/supercollider/supercollider/pull/6323#discussion_r1636417548

grafik

We could fix this for good by adding a pre-commit step to check for excessive newlines on  `.sc` and `.schelp` files.

**Comment by mtmccrea** [comment]
Apologies, I saw your comment about the `:;` error and that there was a follow up correction, I'm not sure how I missed the other review item. I can follow up and address that.

**Comment by mtmccrea** [comment]
Ah I must have passed it over because that conversation was marked as "resolved".

**Comment by capital-G** [comment]
> Apologies, I saw your comment about the `:;` error and that there was a follow up correction, I'm not sure how I missed the other review item. I can follow up and address that.

It is not too important, but could have been applied before merging :) Lets keep it in its current state, I just wanted to mention it :)

**Pull request merged**
